sourcegraph/sourcegraph.proto

```diff
@@ -986,6 +988,22 @@ message ChangesetUpdateOp {
 	UserSpec Author = 8 [(gogoproto.nullable) = false];
 }
 
+message ChangesetMergeOp {
+	// Repo holds the RepoSpec where the Changeset to be merged is located.
+	RepoSpec repo = 1 [(gogoproto.nullable) = false];
+
+	// ID holds the ID of the changeset that is to be merged.
```

Comment:
Suggestion, shorter comment form. Replace "X holds X ..." with juts "X ...". E.g., this comment can be:

```Go
// ID of the changeset that is to be merged.
```